src/blog/lagom-1-5-0-M3.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Lagom 1.5.0-M3 released!
+date: 2018-09-04
+author_github: ignasi35
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.5.0-M3
+---
+
+We are happy to announce the availability of Lagom 1.5.0-M3 (we skipped Lagom 1.5.0-M1 due to an issue with the release process). This new milestone towards the releasing of Lagom 1.5.0 introduces important improvements to support gRPC in Lagom. This release includes Play 2.7.0-M3 and Akka 2.5.16.
+
+Note that this should not be used in production, that Lightbend Orchestration (sbt-reactive-app) does not yet support Lagom 1.5.0-M3, and that ConductR will not be supported for Lagom 1.5.0. Also note that this is an unsupported, pre-release milestone and we don't guarantee API stability or binary compatibility between milestones.
+
+## Highlights
+
+### Updates Play and Akka
+
+- Bump dependencies: Play was bumped to [2.7.0-M3](https://blog.playframework.com/play-2-7-0-m3-released/) and, since Lagom 1.5.0-M2 we've bumped Akka to [2.5.16](https://akka.io/blog/news/2018/08/29/akka-2.5.16-security-fix-released) which also includes [2.5.15](https://akka.io/blog/news/2018/08/24/akka-2.5.15-released). These include work to reduce code duplication between Play and Lagom helping both projects move faster.
```

Comment:
I find it strange to say that Akka 2.5.16 includes 2.5.15. I understand that we you want to make people aware of the release notes for 2.5.15. 

May we should instead add something like: 

> Bump dependencies: Play was bumped to 2.7.0-M3 [(release notes)](https://blog.playframework.com/play-2-7-0-m3-released/) and, since Lagom 1.5.0-M2 we've bumped Akka to 2.5.16 [(release notes)](https://akka.io/blog/news/2018/08/29/akka-2.5.16-security-fix-released). See also [release notes](https://akka.io/blog/news/2018/08/24/akka-2.5.15-released) for 2.5.15.